xarray/test/test_dask.py

```diff
@@ -182,6 +184,7 @@ def test_univariate_ufunc(self):
         v = self.lazy_var
         self.assertLazyAndAllClose(np.sin(u), xu.sin(v))
 
+    @unittest.skip('currently broken in dask, see GH1090')
```

Comment:
Obv not worth changing, but for future pytest-ty-ness: `@pytest.mark.skip(reason="...")`